src/js/sdk.tsx

```diff
@@ -128,6 +130,9 @@ export function init(passedOptions: ReactNativeOptions): void {
     if (options.attachViewHierarchy) {
       defaultIntegrations.push(new ViewHierarchy());
     }
+    if (options.enableCaptureFailedRequests) {
+      defaultIntegrations.push(new HttpClient());
```

Comment:
M: What if `HttpClient` already exists (manually added)? we cannot overwrite the client that the user added manually.

Reply:
I've also added a test in case the behavior would change in the future.